### ISSUE TITLE
Project status

### ISSUE THREAD

**Issue by hamishfagg** [ToonVanEyck/OpenFlap#6]
Hi there,

First of all, I love this approach to making a split flap display. Its the best I've seen of all the open attempts at this.

The readme warns against making one of these at the moment - what are the hurdles preventing this from being ready to build? Are there any major issues still to work out? 

P.S. Have you tried to make a PCB panel with all of the flaps for one display on it? Even as low as JLC prices are, I wonder if they could be lower with a panel.

Thanks for posting this great project!

**Comment by shagadeli** [comment]
I'm also very excited to build my own but holding up for an update!
Congratulations on this project @ToonVanEyck , looking forward for some news

**Comment by ToonVanEyck** [comment]
Thanks for the kind words guys.

This is very much a hobby project going at a hobby project pace. My life is kind of busy at the moment but I'm trying to work on this at least once a week.

At this moment the repo is kind of a mess, so I would recommend not trying to build it at the moment.

1) I need to update the readme.

2) I need to update all the hardware related files. I'm using Fusion 360 witch is cloud based so I need to download the files and put them in the repo manually. I would prefer to use FreeCad and KiCad as they work with local files but I'm not familiar with the software.

3) I need to reimplements the communication protocol and uart driver on the modules as it is not stable and has some bugs which is preventing the OTA updates from working. Once the OTA is stable I would be able to update the firmware of a whole display at once. Which would allow me to develop a bit faster.

4) I have new hardware where I changed from a MOSFET to an H-bridge to drive the motor. This would allow active breaking of the flap wheel. This is something I want to experiment with.

The silk screen has to be the same for all pcb's in a panel so you would need to order at least 48 panels. I did check the pricing at one point and it was cheaper to not panelized. But pricing often changes at JLC so I don't know if that is still the case.

Regards,
Toon
